OktaLoggerAnalytics.podspec

```diff
@@ -0,0 +1,22 @@
+Pod::Spec.new do |s|
+  s.name             = "OktaLoggerAnalytics"
+  s.version          = "1.3.10"
+  s.summary          = "Implementation of Analytics logger destination"
+  s.description      = "Implementation of Analytics logger destination. Requires OktaLogger/Core"
+  s.homepage         = "https://github.com/okta/okta-logger-swift"
+  s.license          = { :type => 'APACHE2', :file => 'LICENSE' }
+  s.author           = { "Okta Developers" => "developer@okta.com" }
+  s.source           = { :git => "https://github.com/okta/okta-logger-swift.git",  :tag => s.version.to_s }
+  s.osx.deployment_target = '10.14'
+  s.ios.deployment_target = '11.0'
+  s.swift_version = '5.0'
+  s.dependency 'SwiftLint'
+  s.static_framework = true
+  
+  s.source_files = [
+  	'Sources/OktaLogger/Analytics/*',
+  	'Sources/OktaLogger/LoggerCore/*.{h,m,swift}'
```

Comment:
Do we need this? You already have `s.dependency 'OktaLogger/Core'` in subspec

Reply:
If I won't include it started failing on pod lib lint with an error: cannot find dependencies